docs/modules/crypto.md

```diff
@@ -36,4 +38,34 @@ hashobj:update("FirstString"))
 hashobj:update("SecondString"))
 digest = hashobj:finalize()
 print(encoder.toHex(digest))
-```
\ No newline at end of file
+```
+
+## crypto.new_hmac()
+
+Create an object for calculating a HMAC (Hashed Message Authentication Code,
+aka "signature"). A HMAC can be used to simultaneously verify both integrity
+and authenticity of data.
+
+The returned object has `update(str)` and `finalize()` functions, for
+streaming data through the hash function, and for finalizing and returning
+the resulting signature.
+
+
+#### Syntax
+`hashobj = crypto.new_hmac(algo, key)`
+
+#### Parameters
+- `algo` the hash algorithm to use, case insensitive string
+- `key` the signing key (may be a binary string)
+
+#### Returns
+Hasher object with `update` and `finalize` functions available.
+
+#### Example
+```lua
+hmac = crypto.new_hmac("SHA1", "top s3cr3t key")
+hmac:update("Some data"))
```

Comment:
Extra `)` here and below.
Same for `crypto.new_hash()` btw.

Reply:
Good catch. Fixed up both of them. That's what I get for copy/pasting stuff!

Feel free to merge when happy :)